src/applications/simple-forms/mock-simple-forms-patterns/arrayBuilder/components/ArrayBuilderCards.jsx

```diff
@@ -302,7 +303,7 @@ ArrayBuilderCards.propTypes = {
  *   nounSingular="employer"
  *   nounPlural="employers"
  *   isIncomplete={item => !item?.name}
- *   editItemBasePathUrl="/array-multiple-page-builder-item-page-1"
+ *   editItemPathUrl="/array-multiple-page-builder-item-page-1"
  * />
  * ```
  *
```

Comment:
for reviewers - be sure to expand the file below if its collapsed - `arrayBuilder.jsx`